pkg/apiserver/server.go

```diff
@@ -56,6 +56,7 @@ func Start() {
 	r.Path("/api/v1/app/airgap").Methods("OPTIONS", "POST", "PUT").HandlerFunc(handlers.UploadAirgapBundle)
 
 	// Implemented handlers
+	r.Path("/api/v1/preflight/{appSlug}/{clusterSlug}/{sequence}/ignore-rbac").Methods("OPTIONS", "POST").HandlerFunc(handlers.IgnorePreflightRBACErrors)
```

Comment:
I'm open to suggestions here for API routes.

Reply:
if we're going to have any other preflight routes, we should add an extra constant path here
unless this router will route `/api/v1/preflight/appslug/clusterslug/sequence/other-preflight` properly?